README.md

```diff
@@ -68,8 +84,6 @@ Once you have my repo cloned and you are in the directory, you can setup the fol
 
 You can do this easily from the command line: `heroku config:set KODI_ADDRESS='your_ip_or_dynamic_address' KODI_PORT='kodi_port' KODI_USERNAME='kodi_username' KODI_PASSWORD='kodi_password' --app app-name-and-number`. Changing of course for your settings. You can also use the settings page on your Heroku app to add these.
 
-You can also alternatively store configuration in a file called `.env` which you need to create yourself from a copy of `.env.wsgi`
-
```

Comment:
Was this removed because it doesn't work?  If it does work, it seems like the easiest thing to document since we wouldn't have to update the example command-line above every time we change the variables.